templates/artist/artist_detail.html

```diff
@@ -24,26 +24,24 @@ <h1 class="header-font">{{ artist.name }}</h1>
     <div class="break">
         <hr>
     </div>
-    <div class="extra-padding">
         {% thumbnail artist.photo.img "500" as thumb %}
```

Comment:
Sweet! Now that the `<div>`'s gone, you can unindent the `{% thumbnail %}` and `<img>` tags.